# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../librsync/01-manpage.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Author: n/a
Descriptioin: Improving manpage.

diff -Naurp librsync.orig/doc/rdiff.1 librsync/doc/rdiff.1
--- librsync.orig/doc/rdiff.1	2008-06-20 11:10:01.000000000 +0000
+++ librsync/doc/rdiff.1	2008-06-20 11:29:58.000000000 +0000
@@ -29,10 +29,17 @@ rdiff \- compute and apply signature-bas
 .PP
 \fBrdiff\fP [\fIoptions\fP] \fBpatch\fP \fIbasis-file delta-file new-file\fP
 .fi
+.SH USAGE
+You can use \fBrdiff\fP to update files, much like \fBrsync\fP does.
+However, unlike \fBrsync\fP, \fBrdiff\fP puts you in control.  There
+are three steps to updating a file: \fBsignature\fP, \fBdelta\fP, and
+\fBpatch\fP.
 .SH DESCRIPTION
 In every case where a filename must be specified, \- may be used
 instead to mean either standard input or standard output as
-appropriate.
+appropriate.  Be aware that if you do this, you'll need to terminate your
+\fIoptions\fP with \-\- or \fBrdiff\fP will think you are passing it
+an empty option.
 .SH "RETURN VALUE"
 0 for successful completion, 1 for environmental problems (file not
 found, invalid options, IO error, etc), 2 for a corrupt file and 3 for
diff -Naurp librsync.orig/doc/librsync.3 librsync/doc/librsync.3
--- librsync.orig/doc/librsync.3	2008-06-21 14:50:55.000000000 +0200
+++ librsync/doc/librsync.3	2008-06-24 10:22:49.000000000 +0200
@@ -53,7 +53,7 @@ scriptable access to rsync functions.
 .PP
 .I rdiff and librsync Manual
 .PP
-\fIhttp://rproxy.sourceforge.net/\fP or \fIhttp://linuxcare.com.au/rproxy/\fP.
+\fIhttp://rproxy.sourceforge.net/\fP.
 .PP
 \fIdraft-pool-rsync\fP
 .SH BUGS
